frontend/.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# environment
+.env
```

Comment:
If we're going to end up using Heroku to deploy I don't think you should use `.env` files -- nothing wrong with them but in the Heroku dashboard there's a location to specifically set env variables that we can use instead.

Reply:
that the frontend can access? the benefit of react is using a `.env` file allows us to access env vars using the `process` variable. if we can pull heroku environment variables down into a frontend/.env file then we can access it in the same way

Reply:
It seems like [this](https://create-react-app.dev/docs/adding-custom-environment-variables/) and other articles suggest you an access it through `process` in the same way. If you don't want to do this or test, then ok LGTM but we should make a note of unifying how we read from env variables later.

Also, I'm pretty sure there has to be _some_ way React can get access to environment variables set on the system, so it's just more uniform to set them all one way (`.env` file, or via Heroku). Since my own changes use Heroku and Heroku can preconfigure the variables, it seems better to me.

Reply:
OK, we can try out just setting environment variables via Heroku since it looks like `process.env` has access to it regardless. This is just a gitignore file so I'm going to leave it in for local development at least